packages/terra-theme-context/src/ThemeContextProvider.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from './ThemeContext';
+
+const propTypes = {
+  children: PropTypes.element.isRequired,
+  theme: PropTypes.string,
+};
+
+const ThemeContextProvider = ({ theme, children }) => (
+  <ThemeContext.Provider value={{ theme }}>
```

Comment:
If the only thing coming through the context, I'd say cut on the object altogether and just pass the string down as the value. If you want to keep the object, you'll want to memoize it.

Reply:
I like the object for future passivity. I'll memoize it.

Reply:
Memoized: https://github.com/cerner/terra-framework/pull/1087/commits/8398f7e62234b6c44502add564933a56987e3a40